shared/src/business/useCases/validateCaseDetailInteractor.js

```diff
@@ -6,9 +7,18 @@ const { CaseQC } = require('../entities/cases/CaseQC');
  * @param {object} applicationContext the application context
  * @param {object} providers the providers object
  * @param {object} providers.caseDetail the case data to validate
+ * @param {object} providers.useCaseEntity the flag indicating what kind of entity to use
```

Comment:
🤔 I might rename `useCaseEntity`. I thought it was like "YUCE CASE entity", not "YOUS case entity" if that makes any sense. Maybe `shouldUseCaseEntity`?